src/RTCPeerConnection.ts

```diff
@@ -157,15 +166,23 @@ export default class RTCPeerConnection extends defineCustomEventTarget(...PEER_C
             desc = null;
         }
 
+        this.pendingLocalDescription = new RTCSessionDescription(desc);
+
+        let localDescriptionResult;
+        try {
+            localDescriptionResult = await WebRTCModule.peerConnectionSetLocalDescription(this._pcId, desc);
+        } finally {
+            this.pendingLocalDescription = null;
```

Comment:
AFAIS the current one is still there after sertting it:

<img width="1782" alt="Screen Shot 2023-07-20 at 16 56 58" src="https://github.com/react-native-webrtc/react-native-webrtc/assets/317464/23034cf2-6480-4bda-ad40-da1686bfdccd">

Reply:
I think we need to reset them when the signalling state goes back to stable. While in 'have-local-offer' it should still be pending.

Reply:
Yep, you're right. I'll rework it.

Reply:
Please check the spec, the are probably implications when a rollback happens.